hazelcast/src/test/java/com/hazelcast/map/impl/query/QueryIndexMigrationTest.java

```diff
@@ -160,7 +160,7 @@ public void testQueryWithIndexesWhileMigrating() throws Exception {
     /**
      * test for issue #359
      */
-    @Test(timeout = MINUTE)
+    @Test
```

Comment:
I thought 2 minutes is the default timeout in Hazelcast test runners, or is this not the case?